pkg/scheduler/api/node_info.go

```diff
@@ -580,3 +586,24 @@ func (ni *NodeInfo) SubGPUResource(pod *v1.Pod) {
 		}
 	}
 }
+
+// getUnhealthyGPUs returns all the unhealthy GPU id.
+func (ni *NodeInfo) getUnhealthyGPUs(node *v1.Node) (unhealthyGPUs map[int]bool) {
```

Comment:
Why returns a map whose value is bool? According to the following logic, although the current data struct also works, it seems a simpler data struct meets the requirement such as array, which may be more memory friendly.

Reply:
ok, fix